app/helpers/responses.js

```diff
@@ -0,0 +1,18 @@
+class Responses {
+  static response(res, message, result) {
```

Comment:
Good work here. however, you can do better in naming, calling `Responses.response` can be a bit confusing... how about naming the method something like `formatResponse` or `handleSuccess`... so calling `responses.formatResponse` makes more sense.

Reply:
Worked on it.